lib/extensions/ar_adapter/ar_dba/postgresql.rb

```diff
@@ -14,26 +14,15 @@ def spid
       end
 
       def client_connections
-        data = select(<<-SQL, "Client Connections")
+        data = select(<<-SQL, "Client Connections").to_a
                       SELECT client_addr   AS client_address
                            , datname       AS database
                            , pid           AS spid
-                           , waiting       AS number_waiting
+                           , waiting       AS is_waiting
```

Comment:
Is this changed on PG 9.4 or newer?  I'm curious if this is something that needs to handle different PG versions.